### ISSUE TITLE
Please Update to 1.20.2 and further

### ISSUE THREAD

**Issue by sternschnaube** [dangbroitsdon/Wall-Jump-Unofficial#44]
Any chances you find time for that? 😄 

I loved the way that your mod is able just to run server-side, a huge plus for gameplay experience.

**Comment by dangbroitsdon** [comment]
this will unfortunately never happen. why? well because in the middle of 2022, a tornado hit my hometown and (excuse my language) fucked my neighborhoods power grid for the year proceeding it. this event inevitably cause many power outages for my computer thereby leading to multiple PC corruptions. it happened so much that when I was getting ready to release the new version, the straw on the camels back broke and (excuse my language again) fucked my SSD, which the project was on. I struggle with depression and it made me super man depressed for the months following it for losing all the work I had put into that update which had a lot of stuff which would most likely take a lot of time to re-replicate. So please, this mod will never be picked up again, because I just really don't want to go through that again (it still happens). If you'd like a suitable replacement, look to AzureDoomC's [fork](https://www.curseforge.com/minecraft/mc-mods/wall-jumped) or jahirtrap's [fork](https://www.curseforge.com/minecraft/mc-mods/wall-jump-txf), and if you don't believe a tornado hit my hometown (for some reason), here is a wikipedia link for [it](https://en.wikipedia.org/wiki/Goshen%2C_Ohio#2022_tornado) (i don't really care about doxxing my location). 

I'm not going to close the issue but I will pin it to let others know these series of events.

TL:DR; Power outage killed my SSD and yes the mod is now abandoned

**Comment by dangbroitsdon** [comment]
Honestly, re-reading it, it just seems like I'm complaining about the state of affairs but realistically I had fun working on this project at that time.
